NastassiaBon/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data_extraction
+  file = File.new('data.txt')
+  file.read
+end
+
+def mean(town, data)
+  return -1.0 unless data.include? town
+
+  serch_town(town, data).sum / 12
+end
+
+def variance(town, data)
+  return -1.0 unless data.include? town
+
+  serch_town(town, data).map { |i| (mean(town, data) - i)**2 }.sum / 12
+end
+
+def serch_town(town, data)
+  data.split("\n").each do |i|
+    if i.include? town
+      string = i.scan(/\d*\.\d/).map(&:to_f)
+      return string
```

Comment:
You don't need to return the string. Ruby already returning the value from the last line
![image](https://user-images.githubusercontent.com/11877985/141689152-6f51ba82-41da-48ad-93bf-5f9c8a1ca20c.png)

Reply:
My bad, you need